### PR TITLE
New Chapter

### PR DESCRIPTION
**Pull request by Doishy**
Intended to be post interaction, pre sideboard guide.
Presents the principles of deck construction without a preset "go to" list.

### CODE REVIEW

File: markdown/chapters/meandeck/deckconstruction.md [hunk 1]
```diff
@@ -0,0 +1,603 @@
...
+<row variant="pile">{{!Underground Sea}} {{!Tropical Island}} {{!Bayou}}</row>
+
+The final land type to discuss (for this section) is your basic land count.
+Traditionally only two to three basics were run normally in a 2:1 ratio of
+Island to Swamp. More recently though some are opting to run only one of each,
+some only the Island and others (as mentioned above) are opting to run no
+basic lands at all. If you want to opt for a more control style build then
+you will probably need to look to increase your basic land count to four or 
+five. Normally you wouldn't consider running a splash basic such as a 
+mountain or forest however in the heavier control builds you might consider
```
Comment (Lutri4eva) [inline markdown/chapters/meandeck/deckconstruction.md:130]: is there another word for "consider", so we dont have to use it twice?

File: markdown/chapters/meandeck/deckconstruction.md [hunk 2]
```diff
@@ -0,0 +1,603 @@
...
+to be more disruptive. Again this should not factor as part of your mana
+requirements and instead be considered as Interaction.
+
+Let's consider some example mana bases for different styles of play for the deck.
+We shall present an aggressive, straight {U}{B} *Turbo* manabase, a more balanced
+Grixis {U}{B}{R} manabase and a control style Esper {W}{U}{B} manabase.
+
+#### {U}{B} *Turbo*
+
+- 4 Polluted Delta
```
Comment (Lutri4eva) [inline markdown/chapters/meandeck/deckconstruction.md:158]: 2 of each fetchland is better diversification

File: markdown/chapters/meandeck/deckconstruction.md [hunk 3]
```diff
@@ -0,0 +1,603 @@
...
+
+If you were to run a splash colour you could replace the Island with the {U}/x dual
+land of the appropriate type here.
+
+#### {U}{B}{R} *Balanced*
+
+- 4 Polluted Delta
+- 3 Scalding Tarn
+- 1 Bloodstained Mire
+- 4 Underground Sea
```
Comment (Lutri4eva) [inline markdown/chapters/meandeck/deckconstruction.md:174]: i'm pretty sure most balaced lists only used/use 3 Useas

File: markdown/chapters/meandeck/deckconstruction.md [hunk 4]
```diff
@@ -0,0 +1,603 @@
...
+
+There is one more main class of proactive disruption in the form of running
+a creature package. The most commonly found card for this is {{Baleful Strix}}
+however other options like {{Uro, Titan of Nature's Wrath}} and {{Sedgemoor
+Witch}} have been tried. The idea behind these is they act as delays to
+your opponent's pressure whilst also providing additional card draws.
+In the case of Uro and Sedgemoor, they can also act as full plan B's
+providing an alternative way to win that isn't Doomsday.
+
+If running Strixes, these normally are swapped out for some mix of
```
Comment (Lutri4eva) [inline markdown/chapters/meandeck/deckconstruction.md:314]: This sentence reads a bit confusing, imo

File: markdown/chapters/meandeck/deckconstruction.md [hunk 5]
```diff
@@ -0,0 +1,603 @@
...
+a Pact of Negation too.
+
+#### {U}{B} *Balanced*
+
+- 4 Force of Will
+- 4 Daze
+- 2 Thoughtseize
+- 3 Duress
+- 1 Flusterstorm
+
```
Comment (Lutri4eva) [inline markdown/chapters/meandeck/deckconstruction.md:418]: Maybe we should use 13 interaction spells in this example.
So we have both ends of the spectrum metioned above.
"When combining the two interaction types, most lists will have around 
a total of eleven to thirteen interaction slots in them"

File: markdown/chapters/meandeck/deckconstruction.md [hunk 6]
```diff
@@ -0,0 +1,603 @@
...
+cantrips and then work out what you wish to replace them with at a later point
+in testing.
+
+## 6. Card Group - Pile Cards
+
+
+### 6.1 Thassa's Oracle
+
+This is probably the most diverse set of cards and might as well be titled 
+"miscellaneous" however there are some important pieces in here that
```
Comment (Lutri4eva) [inline markdown/chapters/meandeck/deckconstruction.md:486]: Maybe we can use this as an introductory sentence and move it from 6.1 to 6

File: markdown/chapters/meandeck/deckconstruction.md (hunk 1 above)
Comment (Doishy) [reply markdown/chapters/meandeck/deckconstruction.md]: Changed.

File: markdown/chapters/meandeck/deckconstruction.md (hunk 2 above)
Comment (Doishy) [reply markdown/chapters/meandeck/deckconstruction.md]: Done

File: markdown/chapters/meandeck/deckconstruction.md (hunk 3 above)
Comment (Doishy) [reply markdown/chapters/meandeck/deckconstruction.md]: Done

File: markdown/chapters/meandeck/deckconstruction.md (hunk 4 above)
Comment (Doishy) [reply markdown/chapters/meandeck/deckconstruction.md]: Amended

File: markdown/chapters/meandeck/deckconstruction.md (hunk 5 above)
Comment (Doishy) [reply markdown/chapters/meandeck/deckconstruction.md]: I think I like the variants listed. All were taken from recent lists.

File: markdown/chapters/meandeck/deckconstruction.md (hunk 6 above)
Comment (Doishy) [reply markdown/chapters/meandeck/deckconstruction.md]: Amended.

**Review by angrybacon — APPROVED** [review]

**Pull request merged**